src/Integration/RequestIntegration.php

```diff
@@ -96,17 +96,18 @@ public static function applyToEvent(self $self, Event $event, ?ServerRequestInte
         }
 
         if ($self->options->shouldSendDefaultPii()) {
-            if ($request->hasHeader('REMOTE_ADDR')) {
-                $requestData['env']['REMOTE_ADDR'] = $request->getHeaderLine('REMOTE_ADDR');
+            $server = $request->getServerParams();
```

Comment:
`$serverParams` would be a better name for this variable imo

Reply:
Done